framework/CHANGELOG.md

```diff
@@ -51,6 +51,10 @@ Yii Framework 2 Change Log
 - Enh #11139: `yii\validators\EachValidator` injects specific attribute value in error message parameters (silverfire)
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh #11254: Added ability to attach RBAC rule using class name (mdmunir)
+- Enh #11285: `yii\base\Security` enhancements (tom--, samdark)
+  - Avoid reading too much from `/dev/urandom` and `/dev/random` not to waste entropy.
```

Comment:
This sentence should not be mine :þ

I don't really know what it means to waste entropy and I certainly can't defend the proposition that this change does wastes less entropy than anything else.